torchmultimodal/modules/layers/transformer.py

```diff
@@ -0,0 +1,327 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
Can you maintain the comment on the top of the file: https://github.com/facebookresearch/multimodal/pull/42/files#diff-88df49f19a5f0b27d93c6c37c2d3238bfefd7cd62373645609dc772a1143b126L7-L8?